### PR TITLE
refactor: multiple changes to Dockerfile and executables

### PR DESCRIPTION
**Pull request by carlosmunoz**
Add ZANATA_HOME environment variable instead of user.home in
standalone.xml
Use curl to download dependencies in the Dockerfile instead of
needing to isntall yum
Remove some dependencies between the two running containers
Change some volume mappings

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-docker-files/12)

### CODE REVIEW

**Comment by carlosmunoz** [comment]
_This has been pair-programmed with @seanf. Assigning him as reviewer for final confirmation._

File: zanata-server/runapp.sh [hunk 1]
```diff
@@ -6,11 +6,16 @@ if [ -n "${ZANATA_VERSION-}" ];then
     ZanataVer=$ZANATA_VERSION
 else
     ## Use the ZANATA_VERSION from Dockerfile, line "ARG ZANATA_VERSION="
-    ZanataVer=$(sed -r -n -e '/^ARG\s+ZANATA_VERSION/ s/^ARG\s+ZANATA_VERSION\s*=\s*(.*)\s*/\1/p' < $ScriptDir/Dockerfile)
+    ZanataVer=latest
```
Comment (seanf) [inline zanata-server/runapp.sh:9]: I don't have a problem with this change, but @definite should take a look at it. The right implementation depends on how we expect people to use this script.

Comment (definite) [reply zanata-server/runapp.sh:9]: The only problem it that if we forget to push the version, it won't have error message. Perhaps we can change comment into 
"Use the latest version (not necessary the version defined in Dockerfile)"

**Review by definite — CHANGES_REQUESTED** [review]

**Review by definite — APPROVED** [review]

**Comment by definite** [comment]
:lgtm:

---

Reviewed 3 of 4 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/12#-:-KYzjccXANMmPA12vMas:bnfp4nl)*

**Comment by carlosmunoz** [comment]
@djansen-redhat Would you mind verifying this one?

**Comment by djansen-redhat** [comment]
@carlosmunoz looks good to me

**Pull request merged**